### DISCUSSION TITLE
Custom domain directory?

### DISCUSSION THREAD

**Discussion by strangeh21** [simple-login/app#2349] [Q&A] [answered]
Hi there, I see no way to create a directory for my own personal domains.
Is this simply not possible?

**Comment by nguyenkims** [comment, marked as answer]
Currently this isn't possible indeed.

**Comment by jenkisan** [comment]
I agree. Having the custom domain directory option ties aliases auto created and in directories to the custom domain should something happen to Simplelogin in the future.
